### PR TITLE
feat(sign): Add ftux tooltip for Box Sign entry point

### PR DESCRIPTION
**Pull request by jstoffan**
box-sign-ftux

### CODE REVIEW

File: src/elements/content-sidebar/SidebarNavSign.tsx [hunk 1]
```diff
@@ -10,17 +12,35 @@ import './SidebarNavSign.scss';
 export type Props = PlainButtonProps & {
     intl: IntlShape;
     status?: string;
+    targetingApi?: {
```
Comment (ConradJChan) [inline src/elements/content-sidebar/SidebarNavSign.tsx:15]: Is there no existing type for this?

File: src/elements/content-sidebar/SidebarNavSign.tsx [hunk 2]
```diff
@@ -10,17 +12,35 @@ import './SidebarNavSign.scss';
...
+        onShow: () => void;
+    };
 };
 
-export function SidebarNavSign({ intl, status, ...rest }: Props) {
+export const PlaceholderTooltip = ({ children }: { children: React.ReactNode }) => children;
+
+export function SidebarNavSign({ intl, status, targetingApi, ...rest }: Props) {
+    const isTargeted = targetingApi && targetingApi.canShow;
+    const FtuxTooltip = isTargeted ? TargetedClickThroughGuideTooltip : PlaceholderTooltip;
```
Comment (ConradJChan) [inline src/elements/content-sidebar/SidebarNavSign.tsx:27]: Is the `PlaceholderTooltip` needed? Seems like if `targetingApi.canShow` is false, then the tooltip should not show already, do we need to not render it also?

File: src/elements/content-sidebar/SidebarNavSign.tsx (hunk 1 above)
Comment (jstoffan) [reply src/elements/content-sidebar/SidebarNavSign.tsx:15]: Yes, but only for Flow.

File: src/elements/content-sidebar/SidebarNavSign.tsx (hunk 2 above)
Comment (jstoffan) [reply src/elements/content-sidebar/SidebarNavSign.tsx:27]: The `TargetedClickThroughGuideTooltip` doesn't check to see if `targetingApi` is defined before trying to use it. I figured that pre-empting its rendering was the safest way to handle it.

**Review by ConradJChan — APPROVED** [review]

**Pull request merged**